common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -948,6 +948,13 @@ def make_course_key(self, org, course, run):
         """
         return CourseLocator(org, course, run)
 
+    def make_course_usage_key(self, course_key):
+        """
+        Return a valid :class:`~opaque_keys.edx.keys.UsageKey` for this modulestore
+        that matches the supplied course_key.
+        """
+        return BlockUsageLocator(course_key, 'course', 'course')
```

Comment:
Well, theoretically, the block-cache should work for library content trees as well.  So from that perspective, it's not out of scope.  I was planning to get this implementation reviewed by someone from the Platform team anyway (including yourself).  So thanks for bringing this up.